cfgov/scripts/tests/test_unpublish_closed_jobs.py

```diff
@@ -176,12 +176,14 @@ def test_job_listing_page_live_if_only_1_of_2_links_closed(
             self, request_mock):
         self.assertTrue(self.page.live)
 
+        def side_effect(*args, **kwargs):
```

Comment:
Since you've made this into a function, I don't think you need to define it twice in each test. You could just move it out of the test and then reuse it.

Reply:
Oh nevermind, I forgot they're slightly different.